DfE.FindInformationAcademiesTrusts/Pages/Trusts/Ofsted/CurrentRatings.cshtml

```diff
@@ -0,0 +1,72 @@
+@page "/trusts/ofsted/current-ratings"
+@using DfE.FindInformationAcademiesTrusts.Extensions
+@model CurrentRatingsModel
+
+@{
+  Layout = "Ofsted/_OfstedLayout";
+}
+
+<section class="govuk-!-margin-top-8 app-table-container" data-testid="ofsted-current-ratings-table">
+  <table class="govuk-table" data-module="moj-sortable-table">
+    <caption class="govuk-table__caption govuk-table__caption--m">Current ratings</caption>
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header" aria-sort="ascending" data-testid="ofsted-current-ratings-school-name-header">
+          School name
+        </th>
+        <th scope="col" class="govuk-table__header govuk-body column-header-ofsted-rating" aria-sort="none" data-testid="ofsted-current-ratings-quality-of-education-header">
```

Comment:
We seem to be overriding the width of most columns on most tables but overrides should be the exception rather than the norm.
I think it would be better to override the width of the School name column and before/after joining column if they need to be specific widths

Reply:
Moved comment to #641